src/Client.php

```diff
@@ -41,6 +41,7 @@ class Client extends Model
      */
     protected $casts = [
         'grant_types' => 'array',
+        'scopes' => 'array',
```

Comment:
i think this potentially breaks older clients not using json arrays but space separated scopes?

Reply:
@axlon

Reply:
I'd assume if you already were using a json column with the same name, you'd have overwritten the casts array and this change would be ignored.

Is this breaking something for you?